generate_outpatients.py

```diff
@@ -165,13 +168,15 @@
         F.col("provider"),
         F.col("age"),
         F.col("sex"),
+        F.col("imd_decile"),
```

Comment:
Do we want both decile and quintile in `su_data.nhp.opa`? I think just quintile is enough - won't adding decile in as well massively increase rows?

Reply:
oh. yes! got carried away.

side note, but we should probably re-aggregate the data inside the model; we can drop out any hrg's that aren't included in the inequalities selection to collapse rows at run time